### PR TITLE
Update target-frameworks.md to detail uap10.0.xxxxx

### PR DESCRIPTION
**Pull request by rrelyea**
@anangaur @rido-min @diverdan92 @karann-msft 
added a line with uap10.0.xxxxx pattern for UAP tfms that we implemented in ~4.4 (VS 15.4)

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[59d6188](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/59d61888dca16bc3ff84fa4575f663705478331f)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[59d6188](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/59d61888dca16bc3ff84fa4575f663705478331f)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[59d6188](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/59d61888dca16bc3ff84fa4575f663705478331f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-uapTFMs/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1310) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902122152530736-1310%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vwOunQwK%2B8PZaY7GGkXKQ18e%2FLvTY%2B8bL0w9uTE4C%2Fs%3D&st=2019-02-12T21%3A48%3A57Z&se=2019-03-15T21%3A53%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[a6e6ab8](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/a6e6ab881d4f415773ec3f88391b60350b843846)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
made a small edit to make the language similar to what we have in the product.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[a6e6ab8](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/a6e6ab881d4f415773ec3f88391b60350b843846)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[a6e6ab8](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1310/commits/a6e6ab881d4f415773ec3f88391b60350b843846)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/target-frameworks.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/rrelyea-uapTFMs/docs/reference/target-frameworks.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/target-frameworks?branch=pr-en-us-1310) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902131945071182-1310%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6KQUt9KRCz7tWFol5DyS2vcb5ArfAp4PPCEq9pEvZmc%3D&st=2019-02-13T19%3A41%3A10Z&se=2019-03-16T19%3A46%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
@rido-min - I don't follow. Where do you see that line?

File: docs/reference/target-frameworks.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Windows Phone (SL) | wp | wp [wp7] |
 Windows Phone (UWP) | | wpa81 |
 Universal Windows Platform | uap | uap [uap10.0] |
 | | | uap10.0 |
+| | | uap10.0.xxxxx  (where 10.0.xxxxx is the target platform min version of the consuming app) |
```
Comment (rrelyea) [inline docs/reference/target-frameworks.md]: shouldn't use app -- likely project would be better.
can be dll or app.

**Pull request merged**